sdks/python/apache_beam/io/parquetio_test.py

```diff
@@ -219,20 +233,34 @@ def test_write_display_data(self):
     write = WriteToParquet(file_name, self.SCHEMA)
     dd = DisplayData.create_from(write)
     expected_items = [
+        DisplayDataItemMatcher(
+            'codec',
+            'none'),
         DisplayDataItemMatcher(
             'schema',
             str(self.SCHEMA)),
+        DisplayDataItemMatcher(
+            'row_group_buffer_size',
+            str(64*1024*1024)),
         DisplayDataItemMatcher(
             'file_pattern',
             'some_parquet_sink-%(shard_num)05d-of-%(num_shards)05d'),
-        DisplayDataItemMatcher(
-            'codec',
-            'none'),
         DisplayDataItemMatcher(
             'compression',
             'uncompressed')]
     hc.assert_that(dd.items, hc.contains_inanyorder(*expected_items))
 
+  def test_sink_transform_int96(self):
+    with tempfile.NamedTemporaryFile() as dst:
+      path = dst.name
+      with self.assertRaises(ArrowInvalid):
+        with TestPipeline() as p:
+          # pylint: disable=expression-not-assigned
```

Comment:
AFAIK, pylint directive only affects its own block. I think, only if it's placed at the top of the file, it works for the whole file. I changed the code to use underscore anyway since disabling some checks by comment generally doesn't look good.